doc/website/blog/2022-07-14-announcing-atlas-lint.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Announcing v0.5.0 with Migration Directory Linting
+authors: rotemtam
+tags: [atlas, lint, ci]
+image: https://blog.ariga.io/uploads/images/posts/v0.5.0/atlas-lint.png
+---
+
+With the release of [v0.5.0](https://github.com/ariga/atlas/releases/tag/v0.5.0), we are
+happy to announce a very significant milestone for the project. While this version includes some
+cool features (such as multi-file schemas) and a [swath](https://github.com/ariga/atlas/compare/v0.4.2...v0.5.0)
+of incremental improvements and bugfixes, there is one feature that we're particularly
+excited about and wanted to share with you in the post. 
+
+As most outages happen directly as a result of a change to a system, Atlas provides users with means to verify the
+safety of planned changes before they happen. The [`sqlcheck`](https://pkg.go.dev/ariga.io/atlas@master/sql/sqlcheck)
+package provides interfaces for analyzing the contents of SQL files to generate insights on the safety of many kinds of
+changes to database schemas. With this package developers may define an `Analyzer` that can be used to diagnose the impact
+of SQL statements on the target database.
+
+This functionality is exposed to CLI users via the `migrate lint` subcommand. By utilizing
+the `sqlcheck` package, Atlas can now check your migration directory for common problems
+and issues. One of the core concepts of practicing DevOps is to continuously verify the
+safety of each change, and to do so in an automated way.
+
+### `atlas migrate lint` in action
+
+Recall that Atlas uses a [dev database](https://atlasgo.io/dev-database) to plan and
+simulate schema changes. Let's start by spinning up a container that will serve as our
+dev database: 
+```text
+docker run --name atlas-db-dev -d -p 3307:3306 -e MYSQL_ROOT_PASSWORD=pass  mysql
+```
+
+Next let's create `schema.hcl`, the HCL file which will contain the desired state of
+our database: 
+
+```hcl title=schema.hcl
+schema "example" {
+}
+table "users" {
+  schema = schema.example
+  column "id" {
+    type = int
+  }
+  column "name" {
+    type = varchar(255)
+  }
+  primary_key {
+    columns = [
+      column.id
+    ]
+  }
+}
+```
+
+To simplify the commands we need to type in this demo, let's create an Atlas
+[project file](https://atlasgo.io/cli/projects) to define a local environment:
+```hcl title=atlas.hcl
+env "local" {
+  src = "./schema.hcl"
+  url = "mysql://root:pass@localhost:3306"
```

Comment:
I'd be confused as to why we have a not spinned up database listening on port 3306 here suddenly.

Reply:
I understand, and I will work on dropping this requirement soon so it doesn't need to be explained.